src/components/SignInForm.jsx

```diff
@@ -117,95 +118,101 @@ const SignIn = (props, routeProps) => {
       <Formik
         initialValues={{
           Email: "",
-          Password: "",
+          Password: ""
         }}
         validationSchema={Yup.object().shape({
           Email: Yup.string()
             .email("Please enter a valid email address.")
             .required("Please enter your email address."),
-          Password: Yup.string().required("Please enter your password."),
+          Password: Yup.string().required("Please enter your password.")
         })}
         onSubmit={async (values, actions, setSubmitting) => {
           await userLogin(values, props, actions);
           actions.setSubmitting(false);
         }}
       >
-        {(props) => {
+        {props => {
           const { errors = {}, touched } = props;
           return (
             <Form>
-              {alertReturnValue || hasPasswordReset ? errorMessage : null}
-              <div
-                className={
-                  props.errors.Email && props.touched.Email
-                    ? "cs-form-control error"
-                    : "cs-form-control"
-                }
-              >
-                <label htmlFor="Email">{SignInPage.EmailLabel}</label>
-                <Field type="email" name="Email" />
-                {/* <ErrorMessage name='msg' className='input-feedback' component='div' />
+              <fieldset className="dg_fieldset">
```

Comment:
This should use the [FieldSet](https://baltimorecounty.github.io/dotgov-components/#fieldset) from dotgov components

Reply:
My task was to convert to select on this pull request. That is why I didn't change other stuff . @tmccoy529 are u woking on those?

Reply:
i have not worked on fieldsets yet. We can make that another PR if need be.

Reply:
I was confused, by how the className of dg_fieldset got there, since it's new.